features/environment.py

```diff
@@ -34,19 +36,22 @@ class UAClientBehaveConfig:
     # the test framework
     boolean_options = ["image_clean", "destroy_instances"]
     str_options = ["reuse_image"]
+    redact_options = ["contract_token"]
 
     # This variable is used in .from_environ() but also to emit the "Config
     # options" stanza in __init__
-    all_options = boolean_options + str_options
+    all_options = boolean_options + str_options + redact_options
```

Comment:
If we also kept "contract_token" in the str_options list, we wouldn't need to add those items to "all_options" here and we could drop this.

Reply:
ok. I've changed it.